app/connectors/testOnly/TestOnlyRouterConnector.scala

```diff
@@ -29,28 +29,40 @@ import scala.xml.NodeSeq
 
 class TestOnlyRouterConnector @Inject()(val http: HttpClient, config: FrontendAppConfig)(implicit ec: ExecutionContext) {
 
-  val Log = Logger(getClass)
+  val Log: Logger = Logger(getClass)
 
   def submitInboundMessage(requestData: NodeSeq, headers: Headers)(implicit headerCarrier: HeaderCarrier): Future[HttpResponse] = {
 
     val routerUrl = s"${config.routerUrl}/messages"
     Log.debug(s"Implicit Headers From Core (Connector): ${headerCarrier.headers.toString()}")
     Log.debug(s"Explicit Headers From Core (Connector): ${headers.headers.toString()}")
 
-//    val newHeaders = headerCarrier
-//      .copy(sessionId = None)
-//      .withExtraHeaders(addHeaders(): _*)
-
     val header = headers.headers.filter(x => x._1 == "X-Message-Sender" || x._1 == "X-Message-Type" || x._1 == "Content-Type")
     Log.debug(s"updated header : $header")
+
     http.POSTString[HttpResponse](routerUrl, requestData.toString, header)
   }
 
   private def addHeaders()(implicit headerCarrier: HeaderCarrier): Seq[(String, String)] = Seq("Content-Type" -> "application/xml")
 
-  def submitOutboundMessage(requestData: NodeSeq, headers: Headers)(implicit headerCarrier: HeaderCarrier): Future[HttpResponse] = {
+  def createArrivalNotificationMessage(requestData: NodeSeq, headers: Headers)(implicit headerCarrier: HeaderCarrier): Future[HttpResponse] = {
 
     val serviceUrl = s"${config.destinationUrl}/movements/arrivals"
+
+    Log.debug(s"Implicit Headers To Core (Connector): ${headerCarrier.headers.toString()}")
+    Log.debug(s"Explicit Headers To Core (Connector): ${headers.headers.toString()}")
+
+    val newHeaders = headerCarrier
+      .copy(authorization = Some(Authorization(headers.get("Authorization").getOrElse(""))))
+      .withExtraHeaders(addHeaders(): _*)
+
+    http.POSTString[HttpResponse](serviceUrl, requestData.toString)(rds = HttpReads.readRaw, hc = newHeaders, ec = ec)
+  }
+
+  def submitUnloadingRemarkMessage(requestData: NodeSeq, arrivalId: String, headers: Headers)(implicit headerCarrier: HeaderCarrier): Future[HttpResponse] = {
```

Comment:
This will post a generic message so the messages are not necessarily an Unloading Remark